src/Test/DatabaseResetter.php

```diff
@@ -37,72 +34,52 @@ public static function isDAMADoctrineTestBundleEnabled(): bool
     public static function resetDatabase(KernelInterface $kernel): void
     {
         $application = self::createApplication($kernel);
-        $registry = $kernel->getContainer()->get('doctrine');
+        $databaseResetter = new ORMDatabaseResetter($application, $kernel->getContainer()->get('doctrine'));
```

Comment:
There is no such things like `doctrine:database:drop` or `doctrine:database:create` in `doctrine/odm`. 
That's why we only use ORM here, and why the interface I've created only refers to "reset schema"